Code/Tools/TestImpactFramework/Runtime/Code/Source/TestImpactClientSequenceReport.cpp

```diff
@@ -225,6 +184,13 @@ namespace TestImpact
         {
         }
 
+        ImpactAnalysisSequenceReport::ImpactAnalysisSequenceReport(
+            DraftingSequenceReportBase&& report, const AZStd::vector<AZStd::string>& discardedTestRuns)
```

Comment:
nit: `discardedTestRuns` could be passed by value and then moved (I believe this is now the norm with constructor parameters such as this as a copy will be made anyway in the initializer list)

Reply:
fixed 👍